src/wired_Maksim.d.ts

```diff
@@ -0,0 +1,44 @@
+interface WiredSearchInput {
+  placeholder: string;
+  disabled: boolean;
+  value: string;
+}
+
+interface WiredSlider {
+  value: number;
+  min: number;
+  max: number;
+}
+
+interface WiredSpinner {
+  spinning: boolean;
+  duration: number;
+  value: string;
```

Comment:
There is no description of value in documentation.
https://github.com/rough-stuff/wired-elements/blob/master/docs/wired-spinner.md#properties